pkg/qemu/imgutil/imgutil.go

```diff
@@ -14,6 +14,18 @@ type Info struct {
 	Format string `json:"format,omitempty"` // since QEMU 1.3
 }
 
+func QcowToRaw(source string, dest string) error {
+	var stdout, stderr bytes.Buffer
+	cmd := exec.Command("qemu-img", "convert", source, dest)
```

Comment:
Needs `-O raw`?

Reply:
Default output format is raw, so didn't add it explicitly. 
Let me know if needed, i can i add as well